PubSub/src/Topic.php

```diff
@@ -94,14 +110,20 @@ class Topic
      * @param string $name The topic name
      * @param bool $encode Whether messages should be base64 encoded.
      * @param array $info {
-     *     A [Topic](https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.topics
-     *
-     *     @type string name The name of the topic.
-     *     @type array $labels Key value pairs used to organize your resources.
-     *     @type string $kmsKeyName The resource name of the Cloud KMS
-     *           CryptoKey to be used to protect access to messages published on this
-     *           topic. The expected format is
-     *           `projects/my-project/locations/kr-location/keyRings/my-kr/cryptoKeys/my-key`.
+     *        A [Topic](https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.topics
+     *
+     *        @type string name The name of the topic.
+     *        @type array $labels Key value pairs used to organize your resources.
+     *        @type string $kmsKeyName The resource name of the Cloud KMS
+     *              CryptoKey to be used to protect access to messages published on this
+     *              topic. The expected format is
+     *              `projects/my-project/locations/kr-location/keyRings/my-kr/cryptoKeys/my-key`.
+     *        @type bool $enableCompression Flag to enable compression subject
+     *              tosize of the message. Set the flag to `true` to enable
+     *              compression. Defaults to `false`.
```

Comment:
```suggestion
     *        @type bool $enableCompression Flag to enable compression of the message.
     *              Set the flag to `true` to enable compression. Defaults to `false`.
```

Reply:
I think we should incorporate the info in `$enableCompression` that this will work in coordination with a message threshold size. What do you feel about this?

How should the sentence be here?

Reply:
I think we need not mention it, but if you really want to nudge users to try setting their own thrwshold, I would go like adding this in the threshold param docblock:
`     * These values are experiementally derived after performance evaluations.`

Reply:
Thanks for this.
Incorporated.